frontend/src/components/LoginCard.vue

```diff
@@ -37,7 +38,7 @@
                     required
                   )
                   v-row
-                    v-col.ml-auto.pt-0(cols="auto")
+                    v-col.ml-auto.pt-0(cols="9")
```

Comment:
I think this would break the responsive behaviour. Can you check once?

Reply:
working fine, checked.